### DISCUSSION TITLE
Learning Equality's Open-Source Community Survey

### DISCUSSION THREAD

**Discussion by MisRob** [learningequality/kolibri#11320] [Community]
Hello friends,

I'm reaching out to invite you to take a  survey we've prepared for the open-source community around Learning Equality's products.

We have received so many valuable contributions, thank you! We'd like to understand all the challenges and joys you experience while contributing to our repositories so that we can keep improving our processes and engagement with the community. We'd love to find out more about how we can support you and acknowledge your contributions in ways that are meaningful to your life and career.

We'd be grateful if you could spend 10-15 min and [take the survey](https://forms.gle/uYaWkneJ6GDFQArA7). Thank you for being generous with your time.

On behalf of Learning Equality's team
Michaela

**Comment by MisRob** [comment]
^ @im-NL @Ghat0tkach @dbnicholson @GarvitSinghal47 @Jaspreet-singh-1032 @pwithnall @Devanshu-Augusty here's the survey we talked about in https://github.com/learningequality/kolibri/discussions/11152

**Comment by MisRob** [comment]
I'd also like to invite contributors with whom we co-operated very recently and therefore weren't mentioned in the original thread. We'd be grateful for your input @nikkuAg @dylanmccall @muditchoudhary @thesujai @ShivangRawat30 and @yash1378

**Comment by MisRob** [comment]
And of course, if there's anyone else cooperating with us whom I didn't mention explicitly, you're very welcome to join

**Comment by MisRob** [comment]
Thank you all for your responses. We're now closing the survey and will start evaluation period planned for December and January.

**Reply by 0xV0YD** [reply]
@MisRob mam i am finally back to open source and in this time i have enhanced my skills by learning django , app development and SEO optimization and now i am ready to contirbute fully to the open source so can i start contributing ?

**Reply by MisRob** [reply]
Thanks for volunteering @yash1378. We currently have contributing opportunities in three repositories. You can see the contributing guidelines including links to issues suitable for contribution for each repository here:

Kolibri: https://github.com/learningequality/kolibri/blob/release-v0.16.x/CONTRIBUTING.md
Kolibri Design System: https://github.com/learningequality/kolibri-design-system/blob/main/CONTRIBUTING.md
Studio: https://github.com/learningequality/studio/blob/unstable/CONTRIBUTING.md

**Reply by 0xV0YD** [reply]
thanx for the guidance and reply mam
